instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/http/HttpSpanNameExtractor.java

```diff
@@ -20,22 +23,34 @@ public final class HttpSpanNameExtractor<REQUEST> implements SpanNameExtractor<R
    * Returns a {@link SpanNameExtractor} which should be used for HTTP requests. HTTP attributes
    * will be examined to determine the name of the span.
    */
+  public static <REQUEST> SpanNameExtractor<REQUEST> create(
+      HttpCommonAttributesGetter<REQUEST, ?> getter, Set<String> knownMethods) {
+    return new HttpSpanNameExtractor<>(getter, knownMethods);
```

Comment:
`knownMethods` is an optional setting; and by convention we usually add builders for optional settings. WDYT about adding an `HttpSpanNameExtractorBuilder`?

Reply:
done.